metricbeat/module/vsphere/datastore/datastore.go

```diff
@@ -61,50 +60,46 @@ func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 }
 
 func (m *MetricSet) Fetch() ([]common.MapStr, error) {
-
 	f := find.NewFinder(m.Client, true)
-	if f == nil {
```

Comment:
I looked at the implementation in `find.NewFinder()` and it's not possible for it to return `nil`.